bin/prettier.js

```diff
@@ -193,7 +191,11 @@ if (stdin) {
       if (write) {
         // Don't write the file if it won't change in order not to invalidate
         // mtime based caches.
-        if (output !== input) {
+        if (output === input) {
+          console.log("[ignore] %s", filename);
```

Comment:
It seems also weird that this is called "ignore". To me, ignore means that i'm not even looking at the file. But here, it's more that nothing changed.